### PR TITLE
improvements: Organisation panel and admin panel UI improvements

### PR DESCRIPTION
**Pull request by BALAJI-BALA**

### CODE REVIEW

File: app/controllers/organisation/settings_controller.rb [hunk 1]
```diff
@@ -43,6 +43,26 @@ def destroy
...
+    respond_to do |format|
+      if request.xhr?
+        format.html { render partial: "organisation/settings/respondents_table" }
+      else
+        format.html
+      end
+    end
+	end
+
+	def destroy_respondents
```
Comment (mikevic) [inline app/controllers/organisation/settings_controller.rb:58]: @BALAJI-BALA - is it possible to destroy a respondent that has already submitted an answer?

File: app/javascript/application/images/copy-link.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (mikevic) [inline app/javascript/application/images/copy-link.svg:1]: @BALAJI-BALA - inside the images folder, perhaps we can create another folder for `icons`? So it's organised better

File: config/routes.rb [hunk 3]
```diff
@@ -84,6 +84,8 @@
         patch "/edit_employee/:user_id", to: "employees#edit_employee", as: "edit_employee"
         post :invite, controller: 'employees'
         delete "deactivate/:user_id", to: "employees#deactivate", as: "deactivate"
+        get :list_respondents
+        delete "destroy_respondents/:user_id", to: "settings#destroy_respondents", as: "destroy_respondents"
```
Comment (mikevic) [inline config/routes.rb:88]: @BALAJI-BALA - can we keep the route as 

`delete /respondents/:user_id`?

File: app/controllers/organisation/settings_controller.rb (hunk 1 above)
Comment (BALAJI-BALA) [reply app/controllers/organisation/settings_controller.rb:58]: @mikevic Yeah bro its possible, should I add condition to allow only delete the non-responded respondent?

Comment (mikevic) [reply app/controllers/organisation/settings_controller.rb:58]: @BALAJI-BALA - yes please.

**Pull request merged**